client/src/app/App.js

```diff
@@ -542,16 +552,63 @@ export class App extends Component {
   }
 
   handleError(error, ...args) {
-
     const {
       onError
     } = this.props;
 
+    const {
+      logEntries
+    } = this.state;
+
+    const {
+      message
+    } = error;
+
     if (typeof onError === 'function') {
       onError(error, ...args);
     }
 
-    // TODO: show error in log
+    this.toggleLog(true);
```

Comment:
Duplicate code (cf. `handleWarning`), can we move adding entry + opening log into a reusable function?

Reply:
👍